scc/src/main.rs

```diff
@@ -118,20 +144,25 @@ fn setup_logger(r6_dir: &Path) {
 
 fn compile_scripts(
     script_dir: &Path,
-    cache_dir: &Path,
+    bundle_path: &Path,
     fallback_cache_dir: Option<&Path>,
     files: &Files,
 ) -> Result<(), Error> {
-    let bundle_path = cache_dir.join(BUNDLE_FILE_NAME);
-    let backup_path = cache_dir.join(BACKUP_FILE_NAME);
+    let backup_path = PathBuf::from(format!("{}{}", bundle_path.to_str().unwrap(), BACKUP_FILE_EXT));
```

Comment:
I'd try to use the `Path` API:
```suggestion
    let backup_path = bundle_path.with_extension(BACKUP_FILE_EXT);
```
and modify `BACKUP_FILE_EXT`:
```rs
const BACKUP_FILE_EXT: &str = "redscripts.bk";
```
and apply the same to timestamp file